process/miniblocks/miniblocksProcessor.go

```diff
@@ -18,10 +18,10 @@ import (
 var log = logger.GetOrCreate("indexer/process/miniblocks")
 
 type miniblocksProcessor struct {
-	importDBMode bool
 	hasher       hashing.Hasher
 	marshalier   marshal.Marshalizer
 	selfShardID  uint32
+	importDBMode bool
```

Comment:
Is there any reason for the order change? If not, i won`t make unnecessary changes.

Reply:
for memory aligment